proto/streamlit/proto/Slider.proto

```diff
@@ -30,6 +30,11 @@ message Slider {
     TIME = 4;
   }
 
+  enum Type {
```

Comment:
According to proto best practices, we should reserve a 0 value with unspecified, not used constant, to keep backward capability when we have separate versions of protobuf files on the server and frontend. 
https://developers.google.com/protocol-buffers/docs/style#enums

I know that we do not follow this everywhere (but we do in some places in our code!), so I think for new enums is a good idea to keep it according to best practice just as part of a style guide. 

although I don't have a strong opinion here